lib/internal/cluster/child.js

```diff
@@ -58,11 +65,13 @@ cluster._getServer = function(obj, options, cb) {
   else
     indexes[indexesKey]++;
 
-  const message = util._extend({
+  const message = Object.assign({
```

Comment:
AFAIK we are still using `util._extend()` instead `Object.assign()` for performance reasons.